test/transaction-test.js

```diff
@@ -1008,6 +1008,22 @@ describe('Transaction', function() {
     assert.strictEqual(transaction.tx_json.Fee, '1000');
   });
 
+  it('Set resubmittable', function() {
+    const tx = new Transaction();
+
+    assert.strictEqual(tx.isResubmittable(), true);
+
+    tx.setResubmittable(false);
+    assert.strictEqual(tx.isResubmittable(), false);
+
+    tx.setResubmittable(true);
+    assert.strictEqual(tx.isResubmittable(), true);
+
+    tx.setResubmittable(false);
+    tx.setResubmittable();
```

Comment:
Can we make it require a boolean to simplify the type annotation that we'll add eventually?